File: Data Structures & Algorithms/sort-colors/submission-5.java
class Solution {
    public void sortColors(int[] nums) {
        
        int[] colors = new int[3];
        int n = nums.length;

        for(int i = 0;i<n;i++)
        {
            colors[nums[i]]++;
        }

        int idx = 0;
        for(int i = 0;i<3;i++)
        {
            while(colors[i] > 0)
            {
                nums[idx++] = i;
                colors[i]--;
            }    
        }
    }
}

File: Data Structures & Algorithms/majority-element/submission-5.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        n = len(nums)
        me,count = 0,0

        for i in range(n):
            if count == 0:
                me = nums[i]
                count = 1
            else:
                if me != nums[i]:
                    count -=1
                else:
                    count += 1
        return me